Usage of `pathlib.Path.exists(follow_symlinks=False)`

### Description

The `pwndbg/commands/integration.py` file uses `dest.exists(follow_symlinks=False)`. This should not be allowed as `follow_symlinks` argument for the `pathlib.Path.exists()` method was officially introduced in Python 3.12 and the project needs to support Python 3.10.

### Steps to reproduce

- Run `./lint.sh`

```
pwndbg/commands/integration.py: note: In function "install_generic_plugin":
pwndbg/commands/integration.py:179: error: Unexpected keyword argument "follow_symlinks" for "exists" of "Path"  [call-arg]
            if dest.exists(follow_symlinks=False):
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

### My setup

Pwndbg:   2025.10.20 build: 7dadfebe9 (Linux)
Python:   3.10.12 (main, Nov  4 2025, 08:48:33) [GCC 11.4.0]

(official docker compose container)

--- a/pwndbg/commands/integration.py
+++ b/pwndbg/commands/integration.py
@@ -176,7 +176,7 @@ def install_generic_plugin(
     print("Deleting old files (if they exist):")
     for _, dest in paths:
         print(f"\t{dest}")
-        if dest.exists(follow_symlinks=False):
+        if dest.exists() or dest.is_symlink():
             if dest.is_symlink() or not dest.is_dir():
                 # Works for regular files and symlinks.
                 # Note that `dest.is_dir()` returns True for

